src/main/scala/scala/xml/quote/quote.scala

```diff
@@ -6,10 +6,7 @@ import scala.xml.quote.internal.QuoteImpl
 package object quote {
   implicit class XmlQuote(ctx: StringContext) {
     object xml {
-      def apply[T](args: T*): scala.xml.Node = macro QuoteImpl.apply[T]
-    }
-    object xmls {
-      def apply[T](args: T*): scala.xml.NodeBuffer = macro QuoteImpl.applySeq[T]
+      def apply[T](args: T*): AnyRef = macro QuoteImpl.apply[T]
```

Comment:
I think we have no choice but to return `AnyRef` here, the two types are completely unrelated.